http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -115,33 +116,39 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
         }
       }
 
-      // Create WSM managed identity if shared app
+      wsmResourceApi <- buildWsmResourceApiClient
+
+      // Create or fetch WSM managed identity if shared app
       wsmManagedIdentityOpt <- app.samResourceId.resourceType match {
-        case SamResourceType.SharedApp =>
-          childSpan("createWsmIdentityResource").use { implicit ev =>
```

Comment:
Could you make sure to keep the childSpan to track the managed identity creation?

Reply:
Nvm, I spoke too fast and it is now built into the `createAzureManagedIdentity` function, all good :)